filter_plugins/insert_kernel_modules.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
```

Comment:
filename: insert_kernel_modules is misleading. Better cpu_constraints.py?

Reply:
Renamed in e522a51